src/screens/contacts/ContactsList.js

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import {Image, StyleSheet, Text, View, FlatList} from 'react-native';
+import {createBottomTabNavigator} from '@react-navigation/bottom-tabs';
+import Logo from '../../common/components/Logo';
+import {allContacts} from '../../api/fakeApiUser';
+import Search from '../../components/Search';
+import AddContact from '../addContact/AddContact';
+import Logout from '../logout/Logout';
+import route from '../../utils/constants/routeConstants';
+const screenOptions = {headerShown: false};
+
+const Tab = createBottomTabNavigator();
+
+export function MyTabs() {
+  return (
+    <Tab.Navigator>
+      <Tab.Screen
+        name={route.CONTACTS_LIST}
+        component={ContactsList}
+        options={screenOptions}
+      />
+      <Tab.Screen
+        name={route.ADD_CONTACT}
+        component={AddContact}
+        options={screenOptions}
+      />
+      <Tab.Screen
+        name={route.LOGOUT}
+        component={Logout}
+        options={screenOptions}
+      />
+    </Tab.Navigator>
+  );
+}
+
+const ContactsList = () => {
+  const renderContact = ({item}) => <ContactItem contact={item} />;
+  return (
+    <>
+      <View style={{paddingHorizontal: 33}}>
+        <Logo />
+        <Search />
+        <FlatList
+          data={allContacts}
+          renderItem={renderContact}
+          keyExtractor={item => item.id}
+        />
+      </View>
+    </>
+  );
+};
+
+const ContactItem = ({contact}) => (
+  <View style={styles.userContainer}>
+    <Image
+      style={styles.profileImage}
+      source={require('../../assets/images/dummy_image.png')}
+    />
+    <View style={styles.userDataContainer}>
+      <Text style={styles.userName}>{contact.name}</Text>
+      <Text style={styles.userEmail}>{contact.email}</Text>
+      <Text style={styles.phone}>{contact.phone}</Text>
+    </View>
+    <Image
+      style={styles.right_arrow}
+      source={require('../../assets/images/right_arrow.png')}
+    />
+  </View>
+);
+
+const styles = StyleSheet.create({
+  userContainer: {
+    backgroundColor: '#D2E4FF',
+    alignItems: 'center',
+    justifyContent: 'center',
+    width: 324,
+    height: 123,
+    borderRadius: 15,
+    marginTop: 30,
+    marginHorizontal: 'auto',
+    flexDirection: 'row',
```

Comment:
I see a common mistake in styling that you are repeating yourself with same colors hex code, why can't you create a simple constant objects for all your colors present in your whole app.